interface.go

```diff
@@ -148,6 +162,12 @@ func HandleInput(key *api.Key, page int) {
 		runCommand("xdg-open " + key.Url)
 	}
 	if key.KeyHandler != "" {
+		var deckInfo api.StreamDeckInfo
+		for i := range sDInfo {
+			if sDInfo[i].Serial == dev.Deck.Serial {
+				deckInfo = sDInfo[i]
```

Comment:
In other places you exit with error if there is no deck found. Will just going ahead here not cause a crash?